### DISCUSSION TITLE
Great work, thank you.

### DISCUSSION THREAD

**Discussion by ariznaf** [LibreScore/app-librescore#25] [General]
Completly agree with your opinion about the evolution of musescore.com.

I had previously uploaded a few scores to share for free, now I cannot upload new ones or upgrade what I have (I have to pay for it) and users cannot download them without paying.

I would delete them from there if I find a open/free platform to share them.

**Comment by PeterNjeim** [comment]
I don't think you and I are on the same page. I'm not sure what opinion you're speaking of, but I'm sure it's not something you agree with.

The platform you are speaking of is what Xmader envisioned, and used to be developed here: https://github.com/LibreScore/LibreScore. However it is now abandoned as it would be entirely illegal to distribute copyrighted sheet music.

If the scores you shared aren't your original work or aren't in the public domain, then you have no right to share them (not just for free, but sharing at all), as no one has the right to download them. You can share them via other means, like hosting your own website or to some third-party file server, but you'd still be breaking the law. MuseScore's problem is requiring an account to download public domain scores, not that it charges money to download copyrighted works.

I'm not sure what your comment has to do with the modern LibreScore suite. The current tools we offer are to download scores without an account, and to save to your storage what MuseScore already provides to you.
